kratos/python_scripts/assign_vector_variable_process.py

```diff
@@ -38,7 +37,7 @@ def __init__(self, Model, settings ):
             "model_part_name"      : "please_specify_model_part_name",
             "variable_name"        : "SPECIFY_VARIABLE_NAME",
             "interval"             : [0.0, 1e30],
-            "value"                : [10.0, "3*t", "x+y"],
+            "value"                : [1.0, 0.0, 0.0],
```

Comment:
Why 1,0,0? Why not 0,0,0?
I don’t see how this solves the problem 
Can you elaborate please?

Reply:
Because all other `assign_vector_*` processes have [1,0,0] as a default. I agree it's not ideal but I opted for standarization.

No strong opinions one way or the other though.

Reply:
I changed them all to [0,0,0] so we get the best of both worlds.

Reply:
Indeed, I think that what we should do is to check `Has("value")` prior to the defaults assignation to throw an error if this is not provided. Then the default value would become meaningless.